Makefile

```diff
@@ -155,18 +155,18 @@ define CROSSBUILD_HELP_INFO
 # cross build components.
 #
 # Args:
-#   WHAT: Component names to be lint check. support: $(CROSSBUILD_COMPONENTS)
+#   WHAT: Component names to be lint check. support: $(BINARIES)
 #         If not specified, "everything" will be cross build.
 #
-# GOARM: go arm value, now support:$(GOARM_VALUES)
-#        If not specified ,default use GOARM=GOARM8
+# GOARM: arm version for go crossbuild, now support:$(GOARM_VALUES)
+#        If not specified, default use ARM=GOARM8
 #
 #
 # Example:
 #   make crossbuild
 #   make crossbuild HELP=y
 #   make crossbuild WHAT=edgecore
-#   make crossbuild WHAT=edgecore GOARM=GOARM7
+#   make crossbuild WHAT=edgecore ARM=GOARM7
```

Comment:
`GOARM` is a golang env value, so use ARM here to avoid conflicting with golang. Besides, if I set `GOARM=GOARM8`, will exit with err like: `Invalid GOARM value. Must be 5, 6, or 7.`

Reply:
But why it works before?  Let me have a try.

Reply:
If I set `GOARM=GOARM8`, will exit with err `Invalid GOARM value. Must be 5, 6, or 7.`, before for armv8, just haven't set the  GOARM value, default is armv8

Reply:
agree, but the code should not crush if we pass the value. I create a simple quick fix for this, pls  let me know whether that make sense to you: https://github.com/kubeedge/kubeedge/pull/3522

Reply:
And how about define the variable as "ARM_VERSION" instead of "ARM"?  so it is more descriptive.

Reply:
With your pr, I think we can still use `GOARM` :) PTAL @chendave